epilogue-processor/src/test/java/edu/wpi/first/epilogue/processor/AnnotationProcessorTest.java

```diff
@@ -59,6 +59,45 @@ public void update(DataLogger dataLogger, Example object) {
     assertLoggerGenerates(source, expectedGeneratedSource);
   }
 
+  @Test
+  void basicOptInLogging() {
```

Comment:
I'd call this something like "implicit" logging. The opt-in here is done on the field or method level, so the class itself is implicitly loggable